packaging/installer/dependencies/alpine.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+# Package tree used for installing netdata on distribution:
+# << Alpine >>
+# supported versions: 3.12, 3.13, 3.14, 3.15, edge
+# shellcheck disable=SC2068,SC2086
```

Comment:
If possible, please prefer per-line shellcheck overrides instead of global ones. That way we can more easily keep track of when we can simply get rid of them.